keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2313,6 +2314,13 @@ limited.
 
 There are likely others.
 
+### Introduce v1beta2
```

Comment:
Let's maybe move it to "Implementation History section" ?

Reply:
done, thanks.